web/themes/custom/hatter/css/styles.css

```diff
@@ -0,0 +1,2 @@
+h1,h2,h3{letter-spacing:-.0125em}.banner-title,.button,.button--outline,.button--outline[type=button],.button--outline[type=submit],.button--primary,.button--primary-outline,.button--primary-outline[type=button],.button--primary-outline[type=submit],.button--primary[type=button],.button--primary[type=submit],.button--secondary,.button--secondary-outline,.button--secondary-outline[type=button],.button--secondary-outline[type=submit],.button--secondary[type=button],.button--secondary[type=submit],.button--tertiary,.button--tertiary[type=button],.button--tertiary[type=submit],button,h5,h6,input[type=button],input[type=reset],input[type=submit]{letter-spacing:.1em}h1,h2,h3{font-weight:700}.banner-title,.button,.button--outline,.button--outline[type=button],.button--outline[type=submit],.button--primary,.button--primary-outline,.button--primary-outline[type=button],.button--primary-outline[type=submit],.button--primary[type=button],.button--primary[type=submit],.button--secondary,.button--secondary-outline,.button--secondary-outline[type=button],.button--secondary-outline[type=submit],.button--secondary[type=button],.button--secondary[type=submit],.button--tertiary,.button--tertiary[type=button],.button--tertiary[type=submit],button,h5,h6,input[type=button],input[type=reset],input[type=submit]{text-transform:uppercase}.footer-nav__list,.primary-nav__list,.social--footer{margin:0;padding:0;list-style:none}.clearfix:after,.container:after,.form-block--half:after,.form-block:after,.form-option-container:after,.nest-wrap:after{content:"";display:table;clear:both}.element-focusable,.element-invisible,.social__item--footer span{position:absolute!important;height:1px;width:1px;overflow:hidden;clip:rect(1px 1px 1px 1px);clip:rect(1px,1px,1px,1px)}.element-focusable:active,.element-focusable:focus{position:static!important;clip:auto;height:auto;width:auto;overflow:auto}
```

Comment:
Did you want to commit this?

Reply:
I'm not sure. I think it was being committed and then it was removed and then I think it got re-added. @zendoodles this should be removed correct?